File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/04.For-Loop/03-additional/07.footballLeague.js
function footballLeague(input) {

    const capacity = Number(input[0]);
    const fensCount = Number(input[1]);
    let index = 2;

    let counterA = 0;
    let counterB = 0;
    let counterV = 0;
    let counterG = 0;

    for (let i = 1; i <= fensCount; i++) {
        let current = input[index++];
        switch (current) {
            case "A": counterA++; break;
            case "B": counterB++; break;
            case "V": counterV++; break;
            case "G": counterG++; break;
        }
    }

    const sectorAPrecentage = counterA / fensCount * 100;
    const sectorBPrecentage = counterB / fensCount * 100;
    const sectorVPrecentage = counterV / fensCount * 100;
    const sectorGPrecentage = counterG / fensCount * 100;
    const allFensPercentage = (counterA + counterB + counterV + counterG) / capacity * 100;

    console.log(`${sectorAPrecentage.toFixed(2)}%`);
    console.log(`${sectorBPrecentage.toFixed(2)}%`);
    console.log(`${sectorVPrecentage.toFixed(2)}%`);
    console.log(`${sectorGPrecentage.toFixed(2)}%`);
    console.log(`${allFensPercentage.toFixed(2)}%`);
}
footballLeague(["76"
    , "10"
    , "A"
    , "V"
    , "V"
    , "V"
    , "G"
    , "B"
    , "A"
    , "V"
    , "B"
    , "B"])

File: Python_Basics/Homeworks-And-Labs/04.for_loop/03.additional/07.football_league.py
capacity = int(input())
fens_count = int(input())

counter_a = 0
counter_b = 0
counter_v = 0
counter_g = 0

for i in range(fens_count):
    current = input()
    if current == 'A':
        counter_a += 1
    elif current == 'B':
        counter_b += 1
    elif current == 'V':
        counter_v += 1
    elif current == 'G':
        counter_g += 1
        
sector_a_percentage = counter_a / fens_count * 100
sector_b_percentage = counter_b / fens_count * 100
sector_v_percentage = counter_v / fens_count * 100
sector_g_percentage = counter_g / fens_count * 100
all_fens_percentage = (counter_a + counter_b + counter_v + counter_g) / capacity * 100

print(f'{sector_a_percentage:.2f}%')
print(f'{sector_b_percentage:.2f}%')
print(f'{sector_v_percentage:.2f}%')
print(f'{sector_g_percentage:.2f}%')
print(f'{all_fens_percentage:.2f}%')
